src/common/legacy_config_opts.h

```diff
@@ -511,6 +511,8 @@ OPTION(mds_damage_table_max_entries, OPT_INT)
 // Maximum increment for client writable range, counted by number of objects
 OPTION(mds_client_writeable_range_max_inc_objs, OPT_U32)
 
+OPTION(mds_adopt_homeless_subtree, OPT_BOOL)
```

Comment:
We don't add options to this file anymore. Please drop this change.

Reply:
> We don't add options to this file anymore. Please drop this change.

done